qutebrowser/browser/urlmarks.py

```diff
@@ -272,12 +272,17 @@ def _parse_line(self, line):
         elif len(parts) == 1:
             self.marks[parts[0]] = ''
 
-    def add(self, url, title):
+    def add(self, url, title, toggle=False):
```

Comment:
I think this should be a keyword-only argument, i.e. do `def add(self, url, title, *, toggle=False):` and adjust the caller to do `toggle=toggle`.